_docs/technical/2017-01-15-cli-options.md

```diff
@@ -0,0 +1,727 @@
+    <br/>This option is using in **development** mode only.
+    <br/>Please make sure you don't use `--flat-distr` option with this one.
+    <br/>Default value: none.
+    <br/>Example: `--bitcoin-distr "(3, 100000)"`
+    <br/>&nbsp;
+35. `--exp-distr`
+    <br/>Enable exponential distribution.
+    <br/>This option is using in **development** mode only.
+    <br/>&nbsp;
+36. `--update-latest-path FILEPATH`
+    <br/>Path to update installer file, which should be downloaded by Update System.
+    <br/>Default value: `update-installer.exe`.
+    <br/>&nbsp;
+37. `--update-with-package`
+    <br/>Enable updating via installer.
+    <br/>&nbsp;
+38. `--monitor-port INT`
+    <br/>Port we use to run web monitor on. Monitor mechanism is a part of [`time-warp-nt` package](https://cardano-docs.iohk.io/technical/protocols/time-warp-nt/).
+    <br/>&nbsp;
+
+Example of `cardano-node` command:
+
+~~~
+stack exec -- cardano-node \
+    --listen 0.0.0.0:12100 \
+    --report-server http://34.146.164.19:5555 \
+    --wallet \
+    --keyfile secret.key \
+    --logs-prefix "logs/abc" \
+    --db-path db-abc \
+    --peer 127.0.0.1:3000/S-GJ1HDZyHpIS_ZOFBlZsipkGNXy-tmCDtY_CvaiywE= \
+    --kademlia-dump-path kademlia-abc.dump
+~~~
+
+## `cardano-web-docs`
+
+An executable `cardano-web-docs` is using for building documentation of `cardano-node` web API (see `cardano-node` above).
+
+`cardano-web-docs` hasn't any options or parameters, it runs and produce `web-api.md` file in the current directory.
+
+## `cardano-launcher`
+
+Please read [Cardano SL Launcher](/technical/launcher/) guide for info about `cardano-launcher` executable.
+
+## `cardano-genupdate`
+
+An executable `cardano-genupdate` is using for generation of update. There's no CLI-options, only 3 mandatory parameters:
+
+~~~
+cardano-genupdate [OLD_DIR] [NEW_DIR] [OUTPUT_TAR_FILE]
+~~~
+
+where:
+
+1. `OLD_DIR` - path to a directory with a program, old version. Path must exist.
+2. `NEW_DIR` - path to a directory with a program, new version. Path must exist.
+3. `OUTPUT_TAR_FILE` - path to output `.tar`-file with diff (based on a difference between old version and new one).
+
+Both directories must have one structure (e.g. they must contain the same files), otherwise `cardano-genupdate` will fail.
+
+Example of `cardano-genupdate` command:
+
+~~~
+stack exec -- cardano-genupdate /tmp/v000 /tmp/v001 /tmp/update.tar
+~~~
+
+**Important**: `cardano-genupdate` uses [`bsdiff`](http://www.daemonology.net/bsdiff/) program, so command `bsdiff` must be available in the `PATH`.
+
+## `cardano-wallet-hs2purs`
+
+An executable `cardano-wallet-hs2purs` is using for generating PureScript types based on Haskell types. We use it during buiding Daedalus wallet.
+
+`cardano-wallet-hs2purs` hasn't any options or parameters, it runs in Cardano SL root directory and produce `.purs` files in `daedalus/src/Generated` subdirectory.
+
+## `cardano-analyzer`
+
+An executable `cardano-analyzer` is...
```

Comment:
You probably left this unfinished to complete it later, but it seems you forgot.